### PR TITLE
fix: Fix entrypoint file name

### LINKED ISSUE #1780: `ftl.Main` changed to `ftl.MainKt` in 21.04.0 breaking scripts and Fladle.
**Describe the bug**
In this PR: https://github.com/Flank/flank/pull/1732/files
`ftl.Main` changed to `ftl.MainKt` which breaks scripts and dependencies which were using the old `ftl.Main` task name.
This breaks all versions of `fladle` with the error message: `Error: Could not find or load main class ftl.Main`

**To Reproduce**
1. Update flank in fladle to 21.04.0 with `flankVersion = '21.04.0'`
2. `./gradlew runFlank`
3. `Error: Could not find or load main class ftl.Main`

See this build scan for the error message.

### PR DESCRIPTION
**Pull request by pawelpasterz**
Fixes #1780 

1. Run `./gradlew flankFullRun`
2. Unit tests pass
3. Follow 'steps to reproduce' in https://github.com/Flank/flank/issues/1780
    1. Update flank in fladle to 21.04.0 with flankVersion = '21.04.0'
    2. `./gradlew runFlank`
    3. task proceeds without error

## Checklist

- [x] Unit tested

### CODE REVIEW

**Comment by pawelpasterz** [comment]
@flank-it

**Comment by pawelpasterz** [comment]
> **Integration tests were triggered at 2021-04-07 05:20:17, you can track progress [here](https://github.com/Flank/flank/actions/runs/724903293)**

Will probably fail on windows...

**Comment by pawelpasterz** [comment]
> **Integration tests failed ❌**
> Windows status failure - Build scan:
> MacOS status failure - Build scan: https://gradle.com/s/oxflqffzklham
> Linux status failure - Build scan: https://gradle.com/s/73vj7ewbp5yxs
> Workflow run https://github.com/Flank/flank/actions/runs/724903293

Ah quota...

File: test_runner/src/test/kotlin/ftl/MainTest.kt [hunk 1]
```diff
@@ -100,4 +105,23 @@ class MainTest {
...
+    @Test
+    fun `flank entrypoint should be ftl_Main`() {
+        // For reference: https://github.com/Flank/flank/issues/1780
+
+        val logFile = root.newFile()
+        val result = ProcessBuilder(
+            "java",
+            "-cp",
+            System.getProperty("java.class.path"),
+            "ftl.Main"
```
Comment (runningcode) [inline test_runner/src/test/kotlin/ftl/MainTest.kt:118]: Nice test. Thanks.

**Review by runningcode — APPROVED** [review]

**Review by piotradamczyk5 — APPROVED** [review]

**Comment by pawelpasterz** [comment]
@flank-it

**Review by Sloox — APPROVED** [review]

**Pull request merged**